src/main/scala/org/broadinstitute/dsde/workbench/avram/util/transformers.scala

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.dsde.workbench.avram.util
+
+import cats.data.{EitherT, ReaderT}
+import cats.effect.IO
+import cats.implicits._
+import org.broadinstitute.dsde.workbench.avram.dependencies.{AvramDependencies, Global}
+import org.broadinstitute.dsde.workbench.avram.util.AvramError._
+
+import scala.concurrent.Future
+
+/**
+  * Defines an effect stack for Avram based on cats monad transformers. See:
+  * https://typelevel.org/cats/datatypes/eithert.html
+  * https://typelevel.org/cats/datatypes/kleisli.html
+  * https://typelevel.org/cats-effect/datatypes/io.html
+  */
+object transformers {
+  // AvramResult is a stack of the effects:
+  // - Reader
+  //   - IO
+  //     - Either
+  type AvramReader[A] = ReaderT[IO, AvramDependencies, A]
+  type AvramResult[A] = EitherT[AvramReader, AvramError, A]
+
+  // Conversions to AvramResult
```

Comment:
Here are all the conversions using cats APIs on the underlying monad transformers.